### DISCUSSION TITLE
PageSpeed Insights: "Diagnose performance issues: unknown error"

### DISCUSSION THREAD

**Discussion by nath-artlist** [GoogleChrome/lighthouse#14505] [Q&A]
Hopefully this is the right place for PageSpeed Insights problems?

It shows "Diagnose performance issues: unknown error" for artlist.io: https://pagespeed.web.dev/report?url=https%3A%2F%2Fartlist.io%2F

I've checked:
- Server logs - we return multiple HTTP 200s to a Lighthouse UA in Google's ASN.
- Running in Chrome's dev tools - no issue.
- Running this repo - no issue.

At a loss for where else to look, any thoughts?

**Comment by connorjclark** [comment]
This looks like a problem with PSI that we should investigate - typically sites with heavy usage of WebGL (https://github.com/GoogleChrome/lighthouse/issues/8557) are problematic for PSI, but this site doesn't seem to use any/a lot of webgl.
